CollAction/Dockerfile

```diff
@@ -1,16 +1,16 @@
-﻿FROM microsoft/dotnet:2.1-sdk AS build
+FROM microsoft/dotnet:2.1-sdk AS build
 RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
 RUN apt-get update && apt-get install -y nodejs build-essential
 WORKDIR /src
 COPY CollAction/CollAction.csproj CollAction/
 RUN dotnet restore CollAction/CollAction.csproj
 COPY . .
 WORKDIR /src/CollAction
-RUN npm install # Needed because the project npm install in production mode won't install the devDependencies (webpack etc.)
+RUN npm install # Needed because the project npm install in production mode won''t install the devDependencies (webpack etc.)
```

Comment:
Why does a quote in the comments need to be escaped?

Reply:
vscode messes up file formatting if it's not there.